src/components/LoanCards/ListLoanCard.vue

```diff
@@ -1,6 +1,9 @@
 <template>
-	<div class="row list-loan-card">
-		<div class="list-loan-card-desktop-column show-for-large large-4 xlarge-4 columns">
+	<div class="row list-loan-card" :class="{ 'tw-rounded': roundedCorners }">
+		<div
+			class="list-loan-card-desktop-column show-for-large large-4 xlarge-4 columns"
+			:class="{ 'tw-rounded-l': roundedCorners }"
```

Comment:
Another way to do this would be to apply `tw-overflow-hidden` alongside `tw-rounded` on the `row list-loan-card` div. But, if does the trick and looks fine across various widths let's roll with it!

Reply:
Ah yup, that's slightly simpler, thanks. I was thinking of not adding any new CSS that could potentially change the existing card behavior, but if we're good adding overflow hidden here, works for me. Done!